scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/executors/ForkEnvironmentScriptExecutor.java

```diff
@@ -29,6 +29,7 @@
 import java.io.Serializable;
 import java.util.Collections;
 import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.concurrent.ConcurrentHashMap'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)